src/index.js

```diff
@@ -242,6 +246,27 @@ class FlashlsHandler {
     return videojs.createTimeRange(seekableStart, seekableEnd);
   }
 
+  media_() {
+    const levels = this.tech_.el_.vjs_getProperty('levels');
+    const level = this.tech_.el_.vjs_getProperty('level');
+    let media;
+
+    if (levels.length) {
+      media = {
+        resolvedUri: Object.values(levels[level].urls)[0],
```

Comment:
You should just be able to use `levels[level].url` here

Reply:
The levels[level].urls is an object which has url value for 0, it has few other key-value pairs in it - fixed and length
urls:
0: "https://playertest.longtailvideo.com/adaptive/eleph-audio/448/chunklist_w370587926_b688000_vo_slen_t64TWFpbg==.m3u8"
fixed: false
length: 1

Reply:
right, but `levels[level]` is an object that has both a `urls` property and `url` property, the `url` property will give you the value of the current active url within `urls`

Reply:
Oh..Yes! I missed that.. I will update it.